_pages/security_advisories.md

```diff
@@ -25,6 +25,30 @@ Likewise you will find when it was fixed and who reported the issue.
 If you have found a security issue in OP-TEE, please send us an email (see
 [Contact]) and then someone from the team will contact you for further discussion.
 The initial email doesn't have to contain any details.
+# November 2020
+
+## Replay Attack Vulnerabilities in RPMB Protocol Applications
+
+Western Digital has identified a couple of security vulnerabilities in the RPMB
+protocol. By doing a man in the middle attack it's possible to get the system
+into a state different to what the legitimate user believes it is. The possible
+types of attacks are downgrade prevention, prevent updates where regardless if
+rollback is enabled or not. The mitigation idea is to always at boot (or during
+first use of RPMB) read the write counter from the RPMB and then send dummy
```

Comment:
> "The possible types of attacks are downgrade prevention, prevent updates where regardless if rollback is enabled or not."

I'm not sure I understand this sentence... Maybe just say "The possible types of attacks are software downgrades"?

Reply:
Will fix ... push an update within a couple of minutes.